stdlib/public/core/OutputStream.swift

```diff
@@ -386,13 +387,17 @@ internal func _print_unlocked<T, TargetStream : TextOutputStream>(
 ///
 /// This function is forbidden from being inlined because when building the
 /// standard library inlining makes us drop the special semantics.
+@_inlineable
```

Comment:
This is inline(never) so it doesn't have to be inlineable either -- you'll just end up emitting useless copies of this into client binaries